### DISCUSSION TITLE
Acteur Bouton de navigation : Bordure

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#778] [Q&A]
Bonjour,

Sur l'acteur Bouton de navigation, la bordure s'affiche obligatoirement même si la propriété borderStyle n'est pas renseignée ("none")
Pour être sure de ne pas avoir de bordure, il faut renseigner la propriété borderWidth à 0px

Est-ce normal ?

**Comment by Mathis554** [comment]
Bonjour,
Ce que tu peux faire, c'est mettre les paramètres suivants
![image](https://github.com/witsa/synapps/assets/174346872/8093a400-d9a0-4d15-9c63-3531b6d606c8)
![image](https://github.com/witsa/synapps/assets/174346872/ee5a49b9-e96e-4574-8704-5f2580b65f0d)
![image](https://github.com/witsa/synapps/assets/174346872/a5199e60-b4b4-4939-827f-fbcec8ffcd84)
![image](https://github.com/witsa/synapps/assets/174346872/159fe5a8-a924-4c74-93e1-ac5378e69cee)
Cela permettra de mettre ta bordure et l'ombre de celui-ci en couleur transparent ce qui rend invisibles la bordure et son ombre.

**Comment by JeremieDENAEGEL** [comment]
Bonjour,

Mettre à 0px au lieu de la valeur d'initialisation de 1px la bordure contourne le bug rencontré.
L'autre méthode pour contourner est certe de changer la couleur en "transparent" mais le décalage des acteurs due à la taille bordure reste présent.

**Comment by Raphwit** [comment]
Bonjour, oui c'est normal, de par son comportement intrinsèque aux boutons, ceux-ci disposent d'une bordure. 
Ce n'est donc pas un "bug" à proprement parlé mais plutôt un comportement initial mal reflété.

Merci pour le retour, nous allons prendre en compte la remarque et réfléchir sur l'avenir de cette bordure pour les boutons 😉
